examples/templates/aws-ecs-container/main.tf

```diff
@@ -34,14 +47,14 @@ resource "aws_ecs_task_definition" "workspace" {
   family = "coder"
 
   requires_compatibilities = ["EC2"]
-  cpu                      = var.cpu
-  memory                   = var.memory
+  cpu                      = data.coder_parameter.cpu.value
+  memory                   = data.coder_parameter.memory.value
   container_definitions = jsonencode([
     {
       name      = "coder-workspace-${data.coder_workspace.me.id}"
       image     = "codercom/enterprise-base:ubuntu"
-      cpu       = 1024
-      memory    = 2048
+      cpu       = data.coder_parameter.cpu.value
+      memory    = data.coder_parameter.memory.value
```

Comment:
`tonumber(...)`? 

(for cpu and memory)

Reply:
do we need to number if we hardcode them?
As you suggested they are needed.
```hcl
cpu       = 1024
memory    = 2048
```

Reply:
I am confused about this. section. do we need hardcoded numbers and why? 
If we can use parameters then we can use `tonumber(...)`

Reply:
I'm saying that we can either leave them as they are (1024, 2048) or replace with `tonumber(data.coder_parameter.cpu.value)` and `tonumber(data.coder_parameter.memory.value )`.

I have no idea why they were hardcoded before. That's probably a bug.

Reply:
ok. I will go with `tonumber(...)` assuming it was a bug before. Thank you!